### ISSUE TITLE
radius-markers: NullPointerException for large radiuses

### ISSUE THREAD

**Issue by Skretzo** [Skretzo/runelite-plugins#2]
## [Radius Markers](../tree/radius-markers/src/main/java/com/radiusmarkers)

When using a too large radius (50 or greater) the [`RadiusMarkerOverlay`](../blob/radius-markers/src/main/java/com/radiusmarkers/RadiusMarkerOverlay.java) will throw a `NullPointerException` in [line`#125`](https://github.com/Skretzo/runelite-plugins/blob/radius-markers/src/main/java/com/radiusmarkers/RadiusMarkerOverlay.java#L121-L125). This is a result of using method `getY()` on `null`, as a `LocalPoint` will be `null` if it is outside the current `Scene`<sup>[1](https://github.com/runelite/runelite/blob/master/runelite-api/src/main/java/net/runelite/api/coords/LocalPoint.java#L66-L85)</sup>.

```java
[Client] WARN  n.r.c.ui.overlay.OverlayRenderer - Error during overlay rendering
java.lang.NullPointerException: null
	at com.radiusmarkers.RadiusMarkerOverlay.renderBox(RadiusMarkerOverlay.java:125)
	at com.radiusmarkers.RadiusMarkerOverlay.drawBox(RadiusMarkerOverlay.java:105)
	at com.radiusmarkers.RadiusMarkerOverlay.render(RadiusMarkerOverlay.java:63)
	at net.runelite.client.ui.overlay.OverlayRenderer.safeRender(OverlayRenderer.java:750)
	at net.runelite.client.ui.overlay.OverlayRenderer.renderOverlays(OverlayRenderer.java:262)
	at net.runelite.client.ui.overlay.OverlayRenderer.renderOverlayLayer(OverlayRenderer.java:196)
	at net.runelite.client.callback.Hooks.drawScene(Hooks.java:430)
	at hz.no(hz.java:33148)
	at cz.gc(cz.java:5279)
	at ou.is(ou.java:10034)
	at ou.is(ou.java:10065)
	at al.ib(al.java:9923)
	at client.gy(client.java:4931)
	at client.aq(client.java:1932)
	at at.b(at.java:406)
	at at.run(at.java:360)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by Skretzo** [comment]
Partially fixed in 37a4ebcb0e6835b762470d71b93908a169dbd2fc.
Large radii are inconsistent when it comes to saving, but throws no error. Look into potential issues with cross-region saving.

**Comment by Skretzo** [comment]
Fixed in 7d99f05c54d9c07e84760c47a294225628206dc3 by changing the radius rendering and removing cross-region reloading.
